synapse/models/infotech.py

```diff
@@ -1255,6 +1259,45 @@ def getModelDefs(self):
                         'doc': 'An array of ATT&CK technique IDs addressed by the mitigation.',
                     }),
                 )),
+                ('it:mitre:attack:campaign', {}, (
```

Comment:
Do we want to add a :references prop to capture the sources of the campaign?

Reply:
I thought about that but I noticed the `mitre.attack.sync` command doesn't use it for any of the existing `it:mitre:attack:*` forms. Instead, it creates a `media:news` node with a `-(refs)>` edge back to the `it:mitre:attack:*`.

I'll add since the other types seem to have it, but be aware that we don't do anything with it. Maybe we should be and this needs a ticket?